client/client.go

```diff
@@ -90,6 +91,9 @@ type (
 	// ConnectionOptions are optional parameters that can be specified in ClientOptions
 	ConnectionOptions = internal.ConnectionOptions
 
+	// Credentials are optional credentials that can be specified in ClientOptions.
+	Credentials = internal.Credentials
```

Comment:
This is considered stable from the SDK perspective right? On Temporal cloud it is not yet AFAIK.

Reply:
I think so unless we're concerned about the API. Self-hosted users could leverage this today instead of the header-providing mechanisms they currently use.